docs/mobility/motion/algorithms.md

```diff
@@ -43,3 +47,11 @@ If unsuccessful, CBiRRT returns an error.
 However, if RRT*-Connect is initially successful, Viam evaluates the path for optimality.
 If the total amount of joint excursion is more than double the minimum possible to go directly to the best Inverse Kinematics solution, then Viam runs CBiRRT to attempt to get a better path than what RRT*-Connect was able to create.
 Viam smooths the two paths, then compares them to one another, and returns the most optimal path.
+
+## Bases
+
+### TP-Space-RRT
+
+This algorithm is based on the research described in the paper: [TP-Space RRT – Kinematic Path Planning of Non-Holonomic Any-Shape Vehicles](https://www.researchgate.net/publication/275584014_TP-Space_RRT_-_Kinematic_Path_Planning_of_Non-Holonomic_Any-Shape_Vehicles). This algorithm is suitable to planning for Bases since it is able to incorporate non-holonomic constraints into the planner directly to only return plans that are kinematically feasible for the base.  One such example of a constraint that this allows us to impose is a minimum turning radius constraint, which is expressed through the Base's Properties method.  
```

Comment:
```suggestion
This algorithm is based on the research described in the paper: [TP-Space RRT – Kinematic Path Planning of Non-Holonomic Any-Shape Vehicles](https://www.researchgate.net/publication/275584014_TP-Space_RRT_-_Kinematic_Path_Planning_of_Non-Holonomic_Any-Shape_Vehicles). This algorithm is suitable to planning for Bases since it is able to incorporate non-holonomic constraints into the planner directly to only return plans that are kinematically feasible for the base.  One such example of a constraint that this allows us to impose is a minimum turning radius constraint, which is expressed through the [Base's](/components/base/) [GetProperties](https://docs.viam.com/components/base/#getproperties) method.  
```